library/unwind/src/lib.rs

```diff
@@ -86,3 +87,8 @@ extern "C" {}
 #[cfg(all(target_vendor = "fortanix", target_env = "sgx"))]
 #[link(name = "unwind", kind = "static", modifiers = "-bundle")]
 extern "C" {}
+
+#[cfg(all(target_os = "windows", target_env = "gnu", target_abi = "llvm"))]
+#[link(name = "unwind", kind = "static", modifiers = "-bundle", cfg(target_feature = "crt-static"))]
+#[link(name = "unwind", cfg(not(target_feature = "crt-static")))]
+extern "C" {}
```

Comment:
Open for discussion: we can always statically link libunwind for these targets.

Reply:
From what I remember from https://github.com/rust-lang/rust/pull/90020 the correct thing to do is more complex, like "link libunwind dynamically if there are dylibs, and otherwise there are options".

Right now we cannot do it properly, that's why the old windows-gnu targets links libgcc through target spec, using `late_link_args_(static,dynamic)`.
Maybe this targets should do the same thing for now, especially given that both specify `crt_static_respected: false` currently.

Reply:
Libunwind is special (in good meaning) here, you can link it statically even for 32-bit DLLs and the exceptions will still work across DLLs*.

* unless somebody also links `libgcc_eh`/`libgcc_s`, in that case no matter what you do 32-bit exceptions are broken.

Reply:
In that case we can always link it statically while `crt_static_respected` is `false` for this target.